manufacturing_system/settings/settings.py

```diff
@@ -165,3 +165,10 @@
 }
 
 COUNTRIES_FLAG_URL = '//localhost:8000/{code}.png'
+
+EMAIL_HOST = 'smtp.yandex.ru'
```

Comment:
I think you can store all of this fields in env, because env more editable, than django settings